### DISCUSSION TITLE
Supported Use Cases for First Release

### DISCUSSION THREAD

**Discussion by ravi-prakash-v** [beckn/DSEP-Specification#48] [General]

**Comment by Vedsaga** [comment]
- Searching for skilled worker kinda of Freelancing.

**Comment by aks30** [comment]
Mentoring solution - 

Enable discovery of mentors across registries

https://docs.google.com/document/d/1Xoir66JZWdSVZqXUjOmkz6X6U293nYqsonhAU84N1zE/edit?usp=sharing

**Comment by sukhpreetssekhon** [comment]
**Brownfield enablement use cases (For education)**

1. Teacher can access lesson plans (filterable wrt grade, subject, competency etc.) from multiple entities
2. Teacher can preview and share relevant content videos from any learning platform with own students 
3. Teacher can view study material and books linked to any learning platform (for ex: DIKSHA, filterable by grade, subject etc.)
4. Teacher can view recommended training videos for self - academic or administrative - on any learning platform
5. Teacher can view a record of content usage across learning platforms
6. Teacher can source attendance information of a student from multiple sources (class, mid day meal, school travel visits)
7. Teacher can view the learning record of the student from multiple learning sources that the student accesses (in class content update, learning platforms)
8. Teacher can view personalized learning recommendations for a student, based on the student's learning record, from multiple partner sources (ngos, content specialists etc)

**Brownfield enablement use cases (For employment - Candidates)**

1. Candidates can register their interest in jobs by sharing their profile and interest areas so that the candidate's profile can be matched with a job vacancy
2. Candidates can auto-generate resumes by providing their basic profile information so that they can share a professional looking one pager that highlights their past experience with a potential employer
3. Candidates receive job vacancy notifications whenever a job requirement is matched with the candidate interest so that the candidate can proactively apply for the job
4. Candidates can submit interest for a potential vacancy by answering questions so that the employer can be updated and get access to the candidate's profile

**Brownfield enablement use cases (For employment - Employers)**

1. Employers can register their job vacancies by mapping to certain standard tags so that the vacancy can be matched with the interest of registered candidates
2. Employers can view and download a list interested candidates along with their profiles so that they can review the candidate's profile and proceed further with the recruitment process
3. Employers can receive periodic statistics of the job vacancy posting so that they can improve the positioning and content in the job vacancies to attract more candidate interest

**Comment by surendrasinghs** [comment]
Few use-cases that were discussed

### Individual can discover training, pay, start learning and get verifiable credentials
Execute a scenario in which an individual can **scan his/her skill key and discover a trusted training institute** in a nearby location. An individual is able to **pay for the training** using an e-RUPI voucher and **start learning**. The training institute should also be able to **issue verifiable credentials** to the individual at the end of the training. This credential is added to the individual’s skill key along with their other certificates.

### An employer can easily issue verifiable credentials
Execute a scenario in which a company **issues a verifiable credential** to an existing (proof of skill) as well as an ex-employee (proof of work).  There should be an ability to **integrate** with the company’s HRMS system for this. This **credential is added** to the individual’s skill key.

### An individual can search for jobs and apply
Execute a scenario in which an individual can **scan his/her skill key** (which already has details of his/her credentials, education, experience) to **search** for relevant jobs and **apply**. All the credentials are **shared** (with user consent) with the companies for consideration.

### An employer can discover qualified candidates 
_Will most likely get descoped as it seems far from reality_

Execute a scenario in which an employer can **discover qualified candidates** across multiple platforms for a particular job opening. An employer should be able to **request for credentials** of shortlisted candidates which they receive post the individual’s consent.

**Comment by sujitrs** [comment]
@surendrasinghs @sukhpreetsamagra @aks30 @Vedsaga Please contribute to Use Cases to be Supported by DSEP Specification at
https://forms.gle/KJ1gG9Mj7R9jnHzJ9

**Comment by sujitrs** [comment]
[Use-Cases received till date](https://github.com/beckn/DSEP-Specification/files/8129970/Use-Cases.xlsx)
